ligo/stablecoin/actions.ligo

```diff
@@ -146,6 +146,91 @@ function debit_from
   end
 } with store with record [ ledger = updated_ledger ]
 
+(*
+ * Converts a transfer item into a safelist transfer item
+ *)
+function convert_to_safelist_transfer
+  (const tp : transfer_param
+  ) : safelist_transfer_item is
+    record
+      [ from_ = tp.0
+      ; to_ = tp.1.0
+      ]
+
+(*
+ * Calls `assert_transfer` of the provided safelist contract using
+ * the provided transfer_descriptors.
+ *)
+function call_assert_transfers
+  ( const ops_in : list(operation)
+  ; const opt_sl_address : option(address)
+  ; const transfer_params : transfer_params
+  ) : list(operation) is block
+  { const operations: list(operation) =
+      case opt_sl_address of
+        Some (sl_address) ->
+          case (Tezos.get_entrypoint_opt ("%assertTransfers", sl_address) : option(contract(safelist_assert_transfers_param))) of
+            Some (sl_caddress) ->
+              Tezos.transaction
+                ( List.map(convert_to_safelist_transfer, transfer_params)
+                , 0mutez
+                , sl_caddress) # ops_in
+            | None -> (failwith ("BAD SAFELIST CONTRACT") : list(operation))
+            end
+        | None -> ops_in
+        end
+  } with operations
+
+(*
+ * Calls `Assert_receivers` of the provided safelist contract using
+ * the provided transfer_descriptors.
+ *)
+function call_assert_receivers
+  ( const opt_sl_address : option(address)
+  ; const receivers : list(address)
+  ) : list(operation) is block
+  {
+    const ops_in : list(operation) = nil
+  ; const operations:list(operation) =
+      case opt_sl_address of
+        Some (sl_address) ->
+          case (Tezos.get_entrypoint_opt ("%assertReceivers", sl_address) : option(contract(safelist_assert_receivers_param))) of
+            Some (sl_caddress) ->
+              Tezos.transaction
+                ( receivers
+                , 0mutez
+                , sl_caddress) # ops_in
+            | None -> (failwith ("BAD SAFELIST CONTRACT") : list(operation))
```

Comment:
I believe these errors are also be specified in `SNAKE_CASE` form

Reply:
Changed